pkg/kube/controllers/extendedstatefulset/pod_mutator.go

```diff
@@ -192,6 +192,13 @@ func replaceVersionInName(name string, version string, offset int) string {
 	return name
 }
 
+// getPodIndex returns ordinal of the Pod
+func getPodIndex(name string) string {
```

Comment:
This function isn't used anywhere, is it?

Reply:
Yes, This also need to be removed.